lighthouse.go

```diff
@@ -1318,7 +1300,7 @@ func (lhh *LightHouseHandler) handleHostUpdateNotification(n *NebulaMeta, fromVp
 }
 
 func (lhh *LightHouseHandler) handleHostPunchNotification(n *NebulaMeta, fromVpnAddrs []netip.Addr, w EncWriter) {
-	//TODO: this is kinda stupid
+	//TODO: this is kinda stupid (WHY?)
```

Comment:
I was frustrated that its possible the lighthouse is communicating with us using a non primary vpn addr which meant we need to compare all `fromVpnAddrs` against all configured lighthouse vpn addrs.

We can remove it, mainly a performance reminder and it'll light up on a cpu profile to remind us.